app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
I would probably place a blank line here to break it up visually. Virtually any `if` statement would qualify for [this](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods) in my opinion. There are number of places like this, but rather than flag them I'm going to suggest going through and trying to spot them and decide which way looks best to you.